sjsonnet/src-js/sjsonnet/Platform.scala

```diff
@@ -1,5 +1,10 @@
 package sjsonnet
+
 import java.io.File
+import java.util
+import java.util.regex.Pattern
```

Comment:
I suppose this means that the `scala-js` implementation won't exactly match RE2 syntax and instead will work according to https://www.scala-js.org/doc/regular-expressions.html . Possibly a corner-case worthy of mention in case we end up writing docs for these functions?